spec/ADAS/ADAS.vspec

```diff
@@ -23,6 +24,42 @@
 #  TerrainMode
 #
 
+ActiveAutonomyLevel:
+  datatype: string
+  type: sensor
+  allowed: [
+    SAE_0, 
+    SAE_1,
+    SAE_2,   
+    SAE_2_DISENGAGING,
```

Comment:
Just a thought, would it be more logical to have `SAE_2_DISENGAGING` between `SAE_1` and `SAE_2`, rather than  between `SAE_2` and `SAE_3`? (and similar for other disengaging values). Reason is to have a logical order from least autonomous to most autonomous

Reply:
Reordered as suggested